lib/apipie/extractor/recorder.rb

```diff
@@ -41,6 +41,10 @@ def analyze_functional_test(test_context)
         end
         @response_data = parse_data(response.body)
         @code = response.code
+        version_key = Apipie.configuration.version_in_http_accept_header
+        if version_key &&  request.env['HTTP_ACCEPT'] && request.env['HTTP_ACCEPT'].index("#{version_key}=")
+          @accept_version = request.env['HTTP_ACCEPT'].split("#{version_key}=").last
```

Comment:
I don't think that there is any guarantee that the version key will be the last part of the accept header. It could have additional '; something=something' after it.